### DISCUSSION TITLE
Possible Solution?

### DISCUSSION THREAD

**Discussion by Titanium-Network-Hack3r** [3kh0/ext-remover#148] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
So, I used the extension thingy for a while and then... I discovered my school unblocked the Linux environment in settings. So I was screwing around with it... I downloaded the Linux environment but then discovered terminal was blocked by Light-speed (Or Go-guardian for other schools). So I did some experimenting and figured out you can download konsole (.deb) file from just a url. So I did that and put the file in the folder called Linux. Once it was done downloading I double clicked on it and it installed and showed the konsole in the icon bar. I then used konsole to download Firefox. Steps are listed below:
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Step 1: Make sure that the Linux dev environment is enabled in settings. It will not work without it being enabled and being able to download. If it is not enabled just wait patiently for these nice people who are working on a solution for the bookmark glitch.

You can find it in settings by going to developers and then at the top it should say linux development enviroment and you want to hit turn on. Mine doesn´t show it because it´s already installed.

![Screenshot 2022-10-26 8 59 44 PM](https://user-images.githubusercontent.com/116772816/198166643-3fa19c37-d2f6-4bb4-80c8-ff35a834bd3b.png)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Step 2: It will ask you to create a username (Just make a random one it doesn´t matter) then once its done installing it will open terminal.
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
step 3: The next thing you want to do is get the konsole .deb file. You can do this by entering this into your URL: 	http://ftp.de.debian.org/debian/pool/main/k/konsole/konsole_20.12.3-1_amd64.deb  

 If it says you don´t have authorization don´t worry. Here is the link to the website where you can download it [https://debian.pkgs.org/11/debian-main-amd64/konsole_20.12.3-1_amd64.deb.html](url)   

If that website is blocked and you have a computer and flash drive. Enter that url save it to the flash drive and then put it into the linux folder. If you are not able to do those 3 options respond to this post with your email and I will e-mail you the file. 

Here is a screenshot of the folder you want to save it in:
![Screenshot 2022-10-26 9 14 54 PM](https://user-images.githubusercontent.com/116772816/198168260-dfaef7d5-cc37-4498-87f1-89a335b56888.png)

---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Step 4: Now that the konsole file is in the Linux folder. Double click on it and install it. Once it´s done installing, go to your apps and look for it. Double click on it and now you have a shell that you can type commands into.
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Step 5 Installing Firefox: There are many different apps you can install but the one I will show you to begin with is firefox. To get it open konsole. 

Type in: sudo apt-get update 

Then type in: sudo apt-get upgrade

Then type sudo apt-get install firefox -esr

Let that finish and you should see a firefox icon. Firefox does not have any website restrictions thus doing basically the same thing as the bookmark glitch. 
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
In conclusion this is only a slight chance that this will help all of you and it was just something to keep you guys happy while you wait for them to come up with a solution to the bookmark glitch.

Hope this helped!
Have a great day!

**Comment by Aka-but-nice** [comment]
If you got Linux why are you here, you're using Linux lightly when you can install Fortnite, COD, and Roblox!

**Reply by Titanium-Network-Hack3r** [reply]
You say that as if those games would run on the chromebook

**Reply by rwg100** [reply]
to make them run, simply light your chromebook on fire and threaten it until it complys. if it doesnt then buy a chromebook, rip out the apu preinstalled, brtually force and i9 12900ks and a rtx 4090 into it, and shut it and die

**Comment by Aka-but-nice** [comment]
And if you got Linux your school is retarded my school won't allow me to add an android device, and you got Linux!

**Reply by Titanium-Network-Hack3r** [reply]
I will admit, my school IT is very understaffed but they are trying their hardest.
